RemoteSettings.Dockerfile

```diff
@@ -29,7 +29,8 @@ RUN pip install ./kinto-remote-settings
 
 # We build the Kinto Admin assets at the specific
 # version specified in `kinto-admin/VERSION`.
-FROM node:21.6.2 as build-admin
+FROM debian:latest as get-admin
```

Comment:
I think it's a bad idea to switch to a `latest` flag. Can we pin to `bookworm`, if not `bookworm-<date>`? It's always annoying when a CI pipeline magically starts failing and you're scrambling trying to figure out why. I'll take some Dependabot noise if it means avoiding silent failures.

I also wonder if we could use the `curl` image here, if that's all we need (well, plus `bash`). Don't know how that image compares with this one as far as how often new tags are published and size.

Reply:
The curl images seem _very_ stripped down and won't have enough to work here. I've switched to `alpine:3` for now as that might be the easiest. But it doesn't include curl. Let me know if you'd rather still use curl as they don't include that by default.

I could also merge this into the `compile` container, but then we're re-pulling the UI every time something else changes.